json-endpoints/jetpack/class.jetpack-json-api-plugins-modify-endpoint.php

```diff
@@ -44,7 +44,11 @@ public function default_action() {
 			if ( $args['active'] ) {
 				return $this->activate();
 			} else {
-				return $this->deactivate();
+				if( $this->current_user_can( 'deactivate_plugins' ) ) {
```

Comment:
I think we should do both capabilities check inside `default_action` or both inside `activate`/`deactivate`

Reply:
I would implement that after we don't care about the wp version any more. 
But maybe we would always care about the version? Since it would stop working for users?

Reply:
Todo: Add a comment why don't need this type of check for activate_plugins.